kolibri/plugins/epub_viewer/assets/src/views/EpubRendererIndex.vue

```diff
@@ -704,7 +704,13 @@
         this.currentSection = this.getCurrentSection(currentLocationStart);
       },
       relocatedHandler(location) {
-        this.sliderValue = location.start.percentage * 100;
+        // Ensures that when we're on the last page, we set the slider value to 100
+        // otherwise, we show the slider % using the start
+        if (location.end.percentage === 100) {
```

Comment:
Should this be `== 100` or `== 1` - I am guessing based on the fact that we multiply it by `100` below that the `location.end.percentage` value is actually a fraction between 0 and 1?

Reply:
Good catch. After looking at what I get in the `location` object, apparently it puts a `atEnd: true` bool flag on it when it's the last page, so I refactored to use that.